node/tests/SharedTests.ts

```diff
@@ -1599,9 +1602,17 @@ export function runBaseTests<Context>(config: {
                 expect(await client.zrange(key, { start: 0, stop: 1 })).toEqual(
                     ["one", "two"],
                 );
-                expect(
-                    await client.zrangeWithScores(key, { start: 0, stop: -1 }),
-                ).toEqual({ one: 1.0, two: 2.0, three: 3.0 });
+                const result = await client.zrangeWithScores(key, {
+                    start: 0,
+                    stop: -1,
+                });
+                const expected = {
+                    one: 1.0,
+                    two: 2.0,
+                    three: 3.0,
+                };
+                expect(Object.keys(expected)).toEqual(Object.keys(result));
```

Comment:
You need to make a function which compares two maps and call it here (and everywhere else in tests)